boards/arm/nucleo_l152re/nucleo_l152re.dts

```diff
@@ -85,6 +89,8 @@
 };
 
 &rtc {
+	clocks = <&rcc STM32_CLOCK_BUS_APB1 0x10000000>,
```

Comment:
Check RM0394 for the stm32l41x/42x/43x/44x/45x/46x  : clocks = <&rcc STM32_CLOCK_BUS_APB1 **0x400**>,